scripts/CEE/drain-reboot-nodes/metadata.yaml

```diff
@@ -0,0 +1,35 @@
+file: script.sh
+name: drain-reboot-worker-node
+description: Drain and Reboot worke node
+author: Fabio Aldana
+allowedGroups:
+  - CEE
+  - SREP
+rbac:
+  clusterRoleRules:
+    - verbs:
+        - "get"
+        - "patch"
+        - "list"
+        - "watch"
+      apiGroups:
+        - ''
+      resources:
+        - "nodes"
+    - verbs:
+        - "get"
+        - "patch"
+        - "list"
+        - "watch"
+        - "create"
+        - "delete"
+      apiGroups:
+        - ''
+      resources:
+        - "pods"
+        - "pods/attach"
```

Comment:
The debug pod in the script is running in namespace `default`, we can limit the permission to namespace `default`. 
```
oc -n default debug node/$WORKER
```

example:  https://github.com/openshift/managed-scripts/blob/651039bbcc4a7aebba47f0a165a4257f177865b0/scripts/SREP/example/metadata.yaml#L16

Reply:
I have added that as recommended.